src/js/outreach/index.js

```diff
@@ -169,9 +169,12 @@ const FormApp = App.extend({
       .fail(({ responseJSON }) => {
         this.showFormSave();
         /* istanbul ignore next: Don't handle non-API errors */
-        if (!responseJSON) return;
+        if (!responseJSON) {
+          this.channel.request('send', 'form:errors', ['Unknown Error']);
```

Comment:
Doesn't this need a `return` after to prevent the next `responseJson.errors` access from showing a warning?

Reply:
I'm mostly just messing with a test that will not fail locally for any reason, but is really flaky on CI